src/infra/core/src/testing/base_repo_harness.rs

```diff
@@ -37,7 +36,7 @@ use crate::DatasetRegistrySoloUnitBridge;
 
 pub struct BaseRepoHarness {
     temp_dir: tempfile::TempDir,
-    catalog: Catalog,
+    catalog: dill::Catalog,
```

Comment:
Can we establish this convention everywhere?
For crates we consider to be external relatively to `"kamu_*"` crates, using fully qualified type names like `dill::Catalog` instead of importing always?

Can something like that be linted as well?